opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -483,7 +483,7 @@ def _translate(self, record: logging.LogRecord) -> LogRecord:
             trace_flags=span_context.trace_flags,
             severity_text=record.levelname,
             severity_number=severity_number,
-            body=record.getMessage(),
+            body=record.msg,
```

Comment:
You mention that "the Body field should be of type 'any' and should not use the str method"
but the record.msg is also a string type, so the difference is just the self.args formating?

       def getMessage(self):
       
        //Return the message for this LogRecord.

        //Return the message for this LogRecord after merging any user-supplied
        //arguments with the message.
        
        msg = str(self.msg)
        if self.args:
            msg = msg % self.args
        return msg

`

Reply:
> You mention that "the Body field should be of type 'any' and should not use the str method" but the record.msg is also a string type, so the difference is just the self.args formating?
> 
> ```
>    def getMessage(self):
>    
>     //Return the message for this LogRecord.
> 
>     //Return the message for this LogRecord after merging any user-supplied
>     //arguments with the message.
>     
>     msg = str(self.msg)
>     if self.args:
>         msg = msg % self.args
>     return msg
> ```

The primary consideration depends on the ultimate purpose of the log. Converting the default log directly into a string is acceptable as it will be required to be presented in a more readable format. However, this approach might not be as "standard" when hoping to aggregate logs and perform subsequent data analysis. In the context of log extraction, it would be more appropriate for the msg to be converted into JSON format or remain unchanged, as it will eventually be transformed into JSON. If the final output JSON data contains a structure that appears similar to JSON but is not, it may confuse users. This is particularly true for operation and maintenance personnel who need to deal with log data in various languages.

Reply:
ok, I see your point, for cases that the msg represents a JSON it will be clear to keep it as a JSON.
Where is the JSON converting occur?
and what about when the msg represents something else but JSON, the expected behavior change?

Reply:
Your comments in this PR are very valuable to understand this change, please include them as comments in the change itself.